aptos-move/e2e-tests/src/data_store.rs

```diff
@@ -118,6 +119,14 @@ impl FakeDataStore {
             StateValue::new_legacy(blob.into()),
         );
     }
+
+    pub fn set_features(&mut self, features: Features) {
```

Comment:
can we unit test this? (that we can deserialize what's set properly?)

Reply:
Added a test!